py3status/modules/wwan_status_nm.py

```diff
@@ -0,0 +1,160 @@
+# -*- coding: utf-8 -*-
+"""
+Display wwan network operator, signal and netgen properties,
+based on ModemManager, NetworkManager and dbus.
+
+Configuration parameters:
+    cache_timeout: How often we refresh this module in seconds.
+        (default 5)
+    consider_3G_degraded: If set to True, only 4G-networks will be
+        considered 'good'; 3G connections are shown
+        as 'degraded', which is yellow by default. Mostly
+        useful if you want to keep track of where there
+        is a 4G connection.
+        (default False)
+    format_down: What to display when the modem is not plugged in.
+        (default 'WWAN: {state} - {operator} {netgen} ({signal}%)')
+    format_error: What to display when modem can't be accessed.
+        (default 'WWAN: {error}')
+    format_up: What to display upon regular connection
+        (default 'WWAN: {state} - {operator} {netgen} ({signal}%)')
+    modem: The modem device to use. If None
+        will use first find modem or
+        use 'busctl introspect org.freedesktop.ModemManager1 \
+            /org/freedesktop/ModemManager1/Modem/0'
+        and read '.EquipmentIdentifier')
+        (default None)
+
+Color options:
+    color_bad: Error or no connection
+    color_degraded: Low generation connection eg 2G
+    color_good: Good connection
+
+Requires:
+    ModemManager
+    NetworkManager
+    pydbus
+
+@author Cyril Levis <levis.cyril@gmail.com>
+
+SAMPLE OUTPUT
+{'color': '#00FF00', 'full_text': u'WWAN: Connected - Bouygues Telecom 4G (19%)'}
+
+off
+{'color': '#FF0000', 'full_text': u'WWAN: Disconnected - Bouygues Telecom 4G (12%)'}
+"""
+
+from pydbus import SystemBus
+
+STRING_WRONG_MODEM = "wrong or any modem"
+STRING_UNKNOW_OPERATOR = "unknow operator"
+
+
+class Py3status:
+    """
+    """
+    # available configuration parameters
+    cache_timeout = 5
+    consider_3G_degraded = False
```

Comment:
We can remove this config when we add `thresholds` for `state` number along with a note. Users just edit thresholds. We also can use `thresholds` for `signal` percent.